system-config/qubes-usbguard.conf

```diff
@@ -0,0 +1,36 @@
+RuleFile=/etc/usbguard/rules.conf
```

Comment:
I guess rpm still allows users to run their own usbguard config overwriting this one?

Reply:
This one is supplied by the distribution; at least on Fedora 35 the default is empty.